cli/cmd/plugin/unmanaged-cluster/cluster/minikube.go

```diff
@@ -222,6 +255,69 @@ func (mkcm *MinikubeClusterManager) PostProviderNotify() []string {
 	return mkcm.logs
 }
 
+// Start attempts to start a stopped minikube cluster. An error is returned when:
+// 1. The cluster is already running.
+// 2. There are issues communicating with minikube.
+// 3. The cluster fails to start.
+func (mkcm *MinikubeClusterManager) Start(c *config.UnmanagedClusterConfig) error {
+	// verify cluster is in a "Stopped" state before attempting to start.
+	kc, err := mkcm.Get(c.ClusterName)
+	if err != nil {
+		return fmt.Errorf("cannot start this cluster. Error occurred retrieving status: %s", err.Error())
+	}
+	if kc.Status != StatusStopped {
+		return fmt.Errorf("cannot start this cluster. The status must be %s, it was %s", StatusStopped, kc.Status)
+	}
+
+	args := []string{
+		"start",
+		"--profile",
+		kc.Name,
+		// specifying the driver is key as the user may have a global driver config with minikube
+		// (via minikube config) and if their global setting is different from the driver used
+		// to start this cluser, creation will fail. Since we lookup the cluster details and know
+		// the driver, we always specify it when running Start.
+		"--driver",
+		kc.Driver,
+	}
+	cmd := exec.Command(minikubeBinName, args...)
+
+	// TODO(joshrosso): starting a cluster can take 1+ minute(s). Could be worth finding a way
+	// to propagate details to the client.
```

Comment:
I know adding `mkcm.logs` and `PostProviderNotify` was a bit of a punt, but I agree, it would be great to offer some kind of way for the providers to set dynamic logs to the `tanzu` client.